rfcs/text/0005-host-metric-fields.md

```diff
@@ -26,6 +26,17 @@ Proposed 7 new fields are:
 <!--
 Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
 -->
+This RFC calls for the addition of host fields to collect basic monitoring metrics from a host or VM such as CPU, network and disk.
+
+| field | type | description |
+| --- | --- | --- |
+| `host.cpu.pct` | scaled_float | Percent CPU used. This value is normalized by the number of CPU cores and it ranges from 0 to 1. |
+| `host.network.in.bytes` | long | The number of bytes received on all network interfaces by the host in a given period of time. |
+| `host.network.in.packets` | long | The number of packets received on all network interfaces by the host in a given period of time. |
+| `host.network.out.bytes` | long | The number of bytes sent out on all network interfaces by the host in a given period of time. |
+| `host.network.out.packets` | long | The number of packets sent out on all network interfaces by the host in a given period of time. |
+| `host.disk.read.bytes` | long | The total number of bytes read successfully in a given period of time. |
```

Comment:
For the `host.disk.[read|write].bytes` metric fields, is the idea to sum the total reads/writes  across all disks mounted on the system for the given time period?

Reply:
It is. The host.disk.* metrics will also be aggregated across all disks, similarly host.network.* metrics are also a sum across all interfaces.